WalletWasabi.Fluent/Models/UI/ApplicationSettings.cs

```diff
@@ -56,7 +59,7 @@ public partial class ApplicationSettings : ReactiveObject
 	// General
 	[AutoNotify] private bool _darkModeEnabled;
 
-	[AutoNotify] private bool _autoCopy;
+	[AutoNotify] private bool _autocopy;
```

Comment:
Why `autocopy`  is wrong while `autoPaste`?

Reply:
There is a mistake in the `UiConfig` in the name. Because I use reflection in this version, i need names to be consistent